.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:**  
+
+**Design doc:**
+
+**Relevant documentation:**
+
+```
+```
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] RFC has been approved. 
+
+**Config**
+
+- [ ] Explicit user action is required to enable this feature (e.g. a config field, config resource, or installation action). 
+
+```
+Link to instructions for enabling
```

Comment:
nit: not sure why its in code block?

Reply:
These code blocks render as text boxes in Markdown. Perhaps there's a better way to create them using markdown. Let me check.

Reply:
I think it's fine to just remove them actually. The template should speak for itself.